fronteerio/canvas-lms
# Ally plugin

This plugin gives instructors feedback on how to make their course
content ore accessible and allows students to download automatically
remediated files.
It adds three endpoints:
- /ally
   Allows our UI utility to figure out whether the plugin has been
   enabled
- /courses/:course_id/ally/sign
   Allows our UI utility to securely sign requests
- /courses/:course_id/ally/proxy
   Allows our UI utility can securely request data from our API

Test plan:
- This requires an Ally account:
- Configure the plugin
- Add the global javascript include
- Verify course files their accessibility are indicates in the files
  and wiki
